src/objects/Player.gd

```diff
@@ -27,10 +26,7 @@ func _ready() -> void:
 	backside_face = FaceState.FACE_4
 	back_face = FaceState.FACE_5
 	bottom_face = FaceState.FACE_6_1
-	$FrontFace.speed_scale = animation_speed
-	$SideFace.speed_scale = animation_speed
-	$TopFace.speed_scale = animation_speed
-	$ExtraFace.speed_scale = animation_speed
+	set_animation_speed()
```

Comment:
Nitpick - can this be `update_animation_speed()` instead since this function isn't taking in a new value to set? It's just updating the values to match what they should be